java_interop/protobuf-triple/java/java-client/src/main/java/org/apache/dubbo/sample/Consumer.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.dubbo.sample;
+
+import org.apache.dubbo.common.constants.CommonConstants;
+import org.apache.dubbo.config.ApplicationConfig;
+import org.apache.dubbo.config.ProtocolConfig;
+import org.apache.dubbo.config.ReferenceConfig;
+import org.apache.dubbo.config.RegistryConfig;
+import org.apache.dubbo.config.bootstrap.DubboBootstrap;
+
+public class ApiConsumer {
+    public static void main(String[] args) {
+        DubboBootstrap bootstrap = DubboBootstrap.getInstance();
+        ReferenceConfig<Greeter> ref = new ReferenceConfig<>();
+        ref.setInterface(Greeter.class);
+        ref.setProtocol(CommonConstants.TRIPLE);
+        ref.setProxy(CommonConstants.NATIVE_STUB);
+        ref.setTimeout(3000);
+        bootstrap.application(new ApplicationConfig("duboo-test"))
+                .registry(new RegistryConfig("zookeeper://127.0.0.1:2181"))
```

Comment:
registry is not necessary for this case, better use `ref.setUrl("tri://ip:port")` instead.